settings.gradle

```diff
@@ -0,0 +1 @@
+rootProject.name = 'hdfs'
```

Comment:
Should we have a different name like `hdfs-mesos` to prevent conflicts with the actual hdfs project?